docs/src/examples/components/SplitButton/Types/index.tsx

```diff
@@ -14,6 +14,11 @@ const SplitButton = () => (
       description="A SplitButton can be formatted to show primary level of emphasis."
       examplePath="components/SplitButton/Types/SplitButtonExamplePrimary"
     />
+    <ComponentExample
+      title="small Split Button"
+      description="small splitbutton"
```

Comment:
Actually, can we follow the size prop that is already in the Button component?

Reply:
Updated it